test/aievec/gemm64_i32_unroll16.mlir

```diff
@@ -125,82 +125,82 @@ module {
 //CHECK-NEXT:        scf.for %arg5 = %c0_2 to %c64_3 step %c16_4 {
 //CHECK-NEXT:          %2 = aievec.upd %arg0[%arg3, %arg5] {index = 0 : i8, offset = 0 : si32} : memref<64x64xi32>, vector<16xi32>
 //CHECK-NEXT:          %3 = aievec.upd %arg1[%arg5, %arg4] {index = 0 : i8, offset = 0 : si32} : memref<64x64xi32>, vector<16xi32>
-//CHECK-NEXT:          %4 = aievec.broadcast %2 {idx = 0 : i8} : vector<16xi32>, vector<16xi32>
+//CHECK-NEXT:          %4 = aievec.broadcast %2 {idx = 0 : si32} : vector<16xi32>, vector<16xi32>
```

Comment:
Same with all these integers with signedness, it's better to keep them signless. Also, consider to maybe use `index` for these attributes.